subworkflows/local/quantify_salmon.nf

```diff
@@ -10,7 +10,6 @@
 include { SALMON_QUANT        } from '../../modules/nf-core/modules/salmon/quant/main' // addParams( options: params.salmon_quant_options   )
 include { SALMON_TX2GENE      } from '../../modules/local/salmon_tx2gene'              // addParams( options: params.salmon_tx2gene_options )
 include { SALMON_TXIMPORT     } from '../../modules/local/salmon_tximport'             // addParams( options: params.tximport_options       )
-include { SALMON_MERGE_COUNTS } from '../../modules/local/salmon_merge_counts'         // addParams( options: params.merge_counts_options   )
```

Comment:
Why did we remove the actual module and import here?

Reply:
It was not used in the workflow. I don't know when it disappeared from the workflow.

Reply:
Looking back through the tags to 3.1, it looks like this module was never used in the workflow from the beginning.

Reply:
Yup. Well spotted 🦅 👀